From 7b0fe668b58bc3d8ad4578e6fce3977db8f996e8 Mon Sep 17 00:00:00 2001
From: Valentin Caron <valentin.caron@foss.st.com>
Date: Wed, 26 Oct 2022 19:06:47 +0200
Subject: [PATCH 129/529] ARM: dts: stm32: add power-domains in stm32mp13x uart
 nodes

On stm32mp13 soc, add UART devices to "pd_core_ret" power domain.

Signed-off-by: Valentin Caron <valentin.caron@foss.st.com>
Change-Id: I3fbfa07cc8f7679db6bdac7a172b26e8787200e4
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/278408
Reviewed-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
Domain-Review: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
---
 arch/arm/boot/dts/stm32mp131.dtsi | 8 ++++++++
 1 file changed, 8 insertions(+)

--- a/arch/arm/boot/dts/stm32mp131.dtsi
+++ b/arch/arm/boot/dts/stm32mp131.dtsi
@@ -175,6 +175,7 @@
 			clocks = <&rcc USART3_K>;
 			resets = <&rcc USART3_R>;
 			wakeup-source;
+			power-domains = <&pd_core_ret>;
 			dmas = <&dmamux1 45 0x400 0x5>,
 			       <&dmamux1 46 0x400 0x1>;
 			dma-names = "rx", "tx";
@@ -188,6 +189,7 @@
 			clocks = <&rcc UART4_K>;
 			resets = <&rcc UART4_R>;
 			wakeup-source;
+			power-domains = <&pd_core_ret>;
 			dmas = <&dmamux1 63 0x400 0x5>,
 			       <&dmamux1 64 0x400 0x1>;
 			dma-names = "rx", "tx";
@@ -201,6 +203,7 @@
 			clocks = <&rcc UART5_K>;
 			resets = <&rcc UART5_R>;
 			wakeup-source;
+			power-domains = <&pd_core_ret>;
 			dmas = <&dmamux1 65 0x400 0x5>,
 			       <&dmamux1 66 0x400 0x1>;
 			dma-names = "rx", "tx";
@@ -250,6 +253,7 @@
 			clocks = <&rcc UART7_K>;
 			resets = <&rcc UART7_R>;
 			wakeup-source;
+			power-domains = <&pd_core_ret>;
 			dmas = <&dmamux1 79 0x400 0x5>,
 			       <&dmamux1 80 0x400 0x1>;
 			dma-names = "rx", "tx";
@@ -263,6 +267,7 @@
 			clocks = <&rcc UART8_K>;
 			resets = <&rcc UART8_R>;
 			wakeup-source;
+			power-domains = <&pd_core_ret>;
 			dmas = <&dmamux1 81 0x400 0x5>,
 			       <&dmamux1 82 0x400 0x1>;
 			dma-names = "rx", "tx";
@@ -276,6 +281,7 @@
 			clocks = <&rcc USART6_K>;
 			resets = <&rcc USART6_R>;
 			wakeup-source;
+			power-domains = <&pd_core_ret>;
 			dmas = <&dmamux1 71 0x400 0x5>,
 			       <&dmamux1 72 0x400 0x1>;
 			dma-names = "rx", "tx";
@@ -350,6 +356,7 @@
 			clocks = <&rcc USART1_K>;
 			resets = <&rcc USART1_R>;
 			wakeup-source;
+			power-domains = <&pd_core_ret>;
 			dmas = <&dmamux1 41 0x400 0x5>,
 			       <&dmamux1 42 0x400 0x1>;
 			dma-names = "rx", "tx";
@@ -363,6 +370,7 @@
 			clocks = <&rcc USART2_K>;
 			resets = <&rcc USART2_R>;
 			wakeup-source;
+			power-domains = <&pd_core_ret>;
 			dmas = <&dmamux1 43 0x400 0x5>,
 			       <&dmamux1 44 0x400 0x1>;
 			dma-names = "rx", "tx";
